code/game/objects/items/corgiclothes.dm

```diff
@@ -0,0 +1,20 @@
+//clothes specifically for Corgis. I had no idea where else to put this so why not make it its own big special thing
+
+/obj/item/corgiclothes
```

Comment:
my snowflake detector is off the charts.
what's wrong with just giving the corgi a real space suit? like every other costume/hat corgis can wear.